/retest
has nothing to do with my changes
```
=== RUN   TestHTTPRouteNotSkippedAfterRouterRestart
    e2e_test.go:283: Deploying InferencePool...
    e2e_test.go:295: Creating HTTPRoute volcano-router-gateway...
    e2e_test.go:310: Restarting kthena-router...
    e2e_test.go:323: Verifying HTTPRoute volcano-router-gateway is processed...
E0424 15:43:49.953688  118856 portforward.go:424] "Unhandled Error" err="an error occurred forwarding 9080 -> 8080: error forwarding port 8080 to pod f35599a324bc8b26c99184c3046a63e2244eaf4dc1e913ddba35256714eb447c, uid : failed to execute portforward in network namespace \"/var/run/netns/cni-215dafbc-7b53-db24-fcd4-98e2011b8fe0\": failed to connect to localhost:8080 inside namespace \"f35599a324bc8b26c99184c3046a63e2244eaf4dc1e913ddba35256714eb447c\", IPv4: dial tcp4 127.0.0.1:8080: connect: connection refused IPv6 dial tcp6 [::1]:8080: connect: connection refused " logger="UnhandledError"
    chat.go:130: Attempt 1/10 failed: Post "http://127.0.0.1:9080/v1/chat/completions": EOF, retrying in 1s...
    chat.go:130: Attempt 2/10 failed: Post "http://127.0.0.1:9080/v1/chat/completions": dial tcp 127.0.0.1:9080: connect: connection refused, retrying in 2s...
    chat.go:130: Attempt 3/10 failed: Post "http://127.0.0.1:9080/v1/chat/completions": dial tcp 127.0.0.1:9080: connect: connection refused, retrying in 4s...
    chat.go:130: Attempt 4/10 failed: Post "http://127.0.0.1:9080/v1/chat/completions": dial tcp 127.0.0.1:9080: connect: connection refused, retrying in 8s...
    chat.go:130: Attempt 5/10 failed: Post "http://127.0.0.1:9080/v1/chat/completions": dial tcp 127.0.0.1:9080: connect: connection refused, retrying in 10s...
    chat.go:130: Attempt 6/10 failed: Post "http://127.0.0.1:9080/v1/chat/completions": dial tcp 127.0.0.1:9080: connect: connection refused, retrying in 10s...
    chat.go:130: Attempt 7/10 failed: Post "http://127.0.0.1:9080/v1/chat/completions": dial tcp 127.0.0.1:9080: connect: connection refused, retrying in 10s...
    chat.go:130: Attempt 8/10 failed: Post "http://127.0.0.1:9080/v1/chat/completions": dial tcp 127.0.0.1:9080: connect: connection refused, retrying in 10s...
    chat.go:130: Attempt 9/10 failed: Post "http://127.0.0.1:9080/v1/chat/completions": dial tcp 127.0.0.1:9080: connect: connection refused, retrying in 10s...
    chat.go:135: 
        	Error Trace:	/home/runner/work/kthena/kthena/test/e2e/utils/chat.go:135
        	            				/home/runner/work/kthena/kthena/test/e2e/utils/chat.go:83
        	            				/home/runner/work/kthena/kthena/test/e2e/utils/chat.go:184
        	            				/home/runner/work/kthena/kthena/test/e2e/utils/chat.go:72
        	            				/home/runner/work/kthena/kthena/test/e2e/router/gateway-inference-extension/e2e_test.go:324
        	            				/opt/hostedtoolcache/go/1.24.0/x64/src/runtime/asm_amd64.s:1700
        	Error:      	Received unexpected error:
        	            	Post "http://127.0.0.1:9080/v1/chat/completions": dial tcp 127.0.0.1:9080: connect: connection refused
        	Test:       	TestHTTPRouteNotSkippedAfterRouterRestart
        	Messages:   	Failed to send HTTP request after retries
--- FAIL: TestHTTPRouteNotSkippedAfterRouterRestart (72.23s)
```

--- a/examples/kthena-router/LLM-Mock-sglang.yaml
+++ b/examples/kthena-router/LLM-Mock-sglang.yaml
@@ -1,6 +1,6 @@
 # SGLang metrics mock deployment for kthena runtime verification.
 # Exposes Prometheus metrics at :30000/metrics compatible with sglang metrics format.
-# Build from aibrix-1/development/app: docker build -t ghcr.io/faust-benchou/sglang-mock:latest -f Dockerfile.sglang .
+# Build from dynamo docker build -t ghcr.io/faust-benchou/dynamo-mocker-sglang:v2026-04-23 -f Dockerfile.sglang .
 
 apiVersion: apps/v1
 kind: Deployment
@@ -18,13 +18,13 @@ spec:
     spec:
       containers:
         - name: llm-engine
-          image: ghcr.io/faust-benchou/sglang-mock:latest
+          image: ghcr.io/faust-benchou/dynamo-mocker-sglang:v2026-04-23
           imagePullPolicy: IfNotPresent
           ports:
             - containerPort: 30000
           env:
             - name: MODEL_NAME
-              value: "meta-llama/Llama-3.1-8B-Instruct"
+              value: "Qwen/Qwen3-8B"
           command:
             - python3
             - sglang_app.py

--- a/examples/kthena-router/LLM-Mock-sglang.yaml
+++ b/examples/kthena-router/LLM-Mock-sglang.yaml
@@ -25,6 +25,3 @@ spec:
           env:
             - name: MODEL_NAME
               value: "Qwen/Qwen3-8B"
-          command:
-            - python3
-            - sglang_app.py

--- a/examples/kthena-router/ModelServer-sglang.yaml
+++ b/examples/kthena-router/ModelServer-sglang.yaml
@@ -9,7 +9,7 @@ spec:
       app: sglang-mock
   workloadPort:
     port: 30000
-  model: "meta-llama/Llama-3.1-8B-Instruct"
+  model: "Qwen/Qwen3-8B"
   inferenceEngine: "SGLang"
   trafficPolicy:
     timeout: 10s

--- a/test/e2e/router/testdata/LLM-Mock-sglang.yaml
+++ b/test/e2e/router/testdata/LLM-Mock-sglang.yaml
@@ -17,13 +17,13 @@ spec:
     spec:
       containers:
         - name: llm-engine
-          image: ghcr.io/faust-benchou/sglang-mock:latest
+          image: ghcr.io/faust-benchou/dynamo-mocker-sglang:v2026-04-23
           imagePullPolicy: IfNotPresent
           ports:
             - containerPort: 30000
           env:
             - name: MODEL_NAME
-              value: "meta-llama/Llama-3.1-8B-Instruct"
+              value: "Qwen/Qwen3-8B"
           command:
             - python3
             - sglang_app.py

--- a/test/e2e/router/testdata/LLM-Mock-sglang.yaml
+++ b/test/e2e/router/testdata/LLM-Mock-sglang.yaml
@@ -24,6 +24,3 @@ spec:
           env:
             - name: MODEL_NAME
               value: "Qwen/Qwen3-8B"
-          command:
-            - python3
-            - sglang_app.py

--- a/test/e2e/router/testdata/ModelServer-sglang.yaml
+++ b/test/e2e/router/testdata/ModelServer-sglang.yaml
@@ -9,7 +9,7 @@ spec:
       app: sglang-mock
   workloadPort:
     port: 30000
-  model: "meta-llama/Llama-3.1-8B-Instruct"
+  model: "Qwen/Qwen3-8B"
   inferenceEngine: "SGLang"
   trafficPolicy:
     timeout: 10s

